### DISCUSSION TITLE
Remove new button

### DISCUSSION THREAD

**Discussion by superXdev** [filamentphp/filament#11493] [Help] [answered]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hi, how I can remove new button at the top of table and keep new button at header of table like this

![IMG_20240220_111405_956](https://github.com/filamentphp/filament/assets/70060487/f4162563-7fb3-4629-b86c-444921e885bc)

**Comment by leandrocfe** [comment, marked as answer]
Go to the `ListReports.php` file and remove this `Actions\CreateAction::make()` in the `getHeaderActions` method.

**Reply by superXdev** [reply]
Thank you!
